x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/event_correlation_rule_suppression.cy.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { getEqlRule } from '../../../../objects/rule';
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { getDetails } from '../../../../tasks/rule_details';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+import {
+  fillAlertSuppressionFields,
+  fillAboutRuleMinimumAndContinue,
+  createRuleWithoutEnabling,
+  skipScheduleRuleAction,
+  continueFromDefineStep,
+  selectAlertSuppressionPerInterval,
+  setAlertSuppressionDuration,
+  selectDoNotSuppressForMissingFields,
+  fillDefineEqlRule,
+  selectEqlRuleType,
+} from '../../../../tasks/create_new_rule';
+
+import {
+  DEFINITION_DETAILS,
+  SUPPRESS_FOR_DETAILS,
+  SUPPRESS_BY_DETAILS,
+  SUPPRESS_MISSING_FIELD,
+  DETAILS_TITLE,
+} from '../../../../screens/rule_details';
+
+const SUPPRESS_BY_FIELDS = ['agent.type'];
+
+describe(
+  'Detection Rule Creation - EQL Rules - With Alert Suppression',
+  {
+    tags: ['@ess', '@serverless'],
```

Comment:
This test should be marked as `@serverless`. Currently, there is no mechanism to enable through configuration FF on MKI environments, so this test is going to start failing. cc @yctercero

Reply:
You mean `@skipInServerless`, right?

Reply:
66008362aea

Reply:
If we can also leave a note or issue reference when we use the skip tags to denote if there's a bug or issue that needs follow up or if it's an intentional skip that does not require any follow up.

Reply:
Yes, our squashing strategy has unfortunately severely reduced the utility of commit messages; the only way to pass along context is through declarative code or code comments :(